app/_de_tools/server.md

```diff
@@ -1,18 +1,23 @@
 ---
 number: 11
-header: "Have your server at home"
-time: "10h"
-skills: 5
-spy_reduction: 2
-incomfort: 4
+header: "Einen eigenen Server zu Hause haben"
+time: "8h"
+skills: 3
+spy_reduction: 4
```

Comment:
Are you sure you can be more secured than a skilled hoster?

Reply:
yes. :-) Yunohost was audited and is very secure. You can install on most of these distro a LE SSL cert in oneclick. And yes, surely more secured, because WE CAN improve security by ourselves, and that is the power of such communities.